server/api/documents.test.js

```diff
@@ -954,6 +955,23 @@ describe("#documents.search", () => {
     expect(res.status).toEqual(401);
     expect(body).toMatchSnapshot();
   });
+
+  it("should save search term, hits and source", async (done) => {
+    const { user } = await seed();
+    await server.post("/api/documents.search", {
+      body: { token: user.getJwtToken(), query: "my term" },
+    });
+
+    setTimeout(async () => {
```

Comment:
These setTimeout's shouldn't be needed, take a look at the other tests for patterns. Other than that I think we're good to go here!

Reply:
I'm sorry, but can you please point an example? the setTimeout is here because we save the search asynchronously, which means that the request ends before the record gets saved into the table.

Reply:
Ah I see, yea this is non-obvious but makes sense now, perhaps just a comment will do then to explain why the timeout is needed.

Reply:
got it